### ISSUE TITLE
Support for SLES

### ISSUE THREAD

**Issue by bernardoVale** [petems/puppet-install-shell#43]
It will be very nice if this plugin could support SLES.

```
13:59:23 -0500 CRIT: Please file a bug report at https://github.com/petems/puppet-install-shell/
13:59:23 -0500 CRIT:
13:59:23 -0500 CRIT: Version: 3.7.4
13:59:23 -0500 CRIT: Platform: sles
13:59:23 -0500 CRIT: Platform Version: 11
13:59:23 -0500 CRIT: Machine: x86_64
13:59:23 -0500 CRIT: OS: Linux
```

**Comment by petems** [comment]
Hi @bernardoVale! I've literally no SLES/SUSE experience to be honest. If I was given a pointer to how it's done currently I could give it a go? There seems to be a split between doing package installs with RPM and Zypper, which one is the more common use?

**Comment by bernardoVale** [comment]
@petems zypper it's more common but you'll have to add a lot of repos. I'm trying to make a simple bash script with the pre requisites to install puppet on SLES 11 SP3. After that I can share it here to be a starting point to puppet-install plugin.

**Comment by petems** [comment]
@bernardoVale That would be awesome, thanks! :smile:

**Comment by gesinn-it** [comment]
@bernardoVale @petems any news on that? Would be cool to have SLES as additional OS.

**Comment by bernardoVale** [comment]
@gesinn-it , Unfortunately I couldn't make any progress with that. It seems that you need SLES SDK repository for the pre-reqs and to use SLES SDK you need a SLES License witch a don't have.

**Comment by gesinn-it** [comment]
@bernardoVale I'm currently working with OpenSuse:

`config.vm.box = "opensuse/openSUSE-42.1-x86_64"`

**Comment by petems** [comment]
SLES support fixed in #98
